README.md

```diff
@@ -43,105 +62,69 @@ vim environments.yaml # use the files you just created
-kubectl delete mutatingwebhookconfigurations prometheus-admission
-kubectl delete ValidatingWebhookConfiguration prometheus-admission
-```
-
-### kafka-init
-This service will create schemes in Kafka and it should do it before other services access data from it. If order hasn't been preserved you should remove Kafka and initialize again:
-```
-helm del --purge cp-zookeeper cp-kafka
-kubectl delete pvc datadir-0-cp-kafka-{0,1,2} datadir-cp-zookeeper-{0,1,2} datalogdir-cp-zookeeper-{0,1,2}
+The final preparation step is to create a certificate for Management Portal to sign requests. 
+```shell
+./bin/keystore-init
 ```
 
-### Uninstall
-If you want to remove the Radar-base from your cluster you need set all of your `RADAR_INSTALL_*` variables in `.env` file to `false` and then run the `helmfile sync --concurrency 1` command to delete the charts after that you need to run following commands to remove all of the traces of the installation:
-```
-kubectl delete crd prometheuses.monitoring.coreos.com prometheusrules.monitoring.coreos.com servicemonitors.monitoring.coreos.com alertmanagers.monitoring.coreos.com podmonitors.monitoring.coreos.com alertmanagerconfigs.monitoring.coreos.com probes.monitoring.coreos.com thanosrulers.monitoring.coreos.com
-kubectl delete psp kube-prometheus-stack-alertmanager kube-prometheus-stack-grafana kube-prometheus-stack-grafana-test kube-prometheus-stack-kube-state-metrics kube-prometheus-stack-operator kube-prometheus-stack-prometheus kube-prometheus-stack-prometheus-node-exporter
-kubectl delete mutatingwebhookconfigurations prometheus-admission
-kubectl delete ValidatingWebhookConfiguration prometheus-admission
-
-kubectl delete crd certificaterequests.cert-manager.io certificates.cert-manager.io challenges.acme.cert-manager.io clusterissuers.cert-manager.io issuers.cert-manager.io orders.acme.cert-manager.io
-kubectl delete pvc --all
-kubectl -n cert-manager delete secrets letsencrypt-prod
-kubectl -n default delete secrets radar-base-tls
-kubectl -n monitoring delete secrets radar-base-tls
-
-kubectl delete crd cephblockpools.ceph.rook.io  cephclients.ceph.rook.io cephclusters.ceph.rook.io cephfilesystems.ceph.rook.io cephnfses.ceph.rook.io cephobjectstores.ceph.rook.io cephobjectstoreusers.ceph.rook.io volumes.rook.io
-kubectl delete psp 00-rook-ceph-operator
+#### Install
+Finally you can start installing the RADAR-Base stack. Having `--concurrency 1` will make installation slower but it is necessary because some components such as `kube-prometheus-stack` and `kafka-init` (aka `catalog-server`) should be installed in their specified order. If you've forgotten to use this flag the installation might not be successful and you should follow [Uninstallation](https://github.com/RADAR-base/RADAR-Kubernetes/tree/dev#uninstall) steps to clean up the Kubernetes cluster before you can try again.
```

Comment:
Does it work to replace
`https://github.com/RADAR-base/RADAR-Kubernetes/tree/dev#uninstall` with `#uninstall`? Then it becomes a relative location.